packages/discord.js/src/util/Formatters.js

```diff
@@ -20,180 +21,510 @@ const {
   userMention,
 } = require('@discordjs/builders');
 
+let deprecatedBlockQuote = false;
+let deprecatedBold = false;
+let deprecatedChannelMention = false;
+let deprecatedCodeBlock = false;
+let deprecatedFormatEmoji = false;
+let deprecatedHideLinkEmbed = false;
+let deprecatedHyperlink = false;
+let deprecatedInlineCode = false;
+let deprecatedItalic = false;
+let deprecatedQuote = false;
+let deprecatedRoleMention = false;
+let deprecatedSpoiler = false;
+let deprecatedStrikethrough = false;
+let deprecatedTime = false;
+let deprecatedUnderscore = false;
+let deprecatedUserMention = false;
```

Comment:
use https://nodejs.org/api/util.html#utildeprecatefn-msg-code instead

Reply:
Should be done with 1ce3eba22752193e09fcbd52b7b944bd5ccf8f6c.